docs/source/cpp/compute.rst

```diff
@@ -197,6 +197,10 @@ an ``Invalid`` :class:`Status` when overflow is detected.
 +--------------------------+------------+--------------------+---------------------+
 | subtract_checked         | Binary     | Numeric            | Numeric             |
 +--------------------------+------------+--------------------+---------------------+
+| divide                   | Binary     | Numeric            | Numeric             |
```

Comment:
Please keep this table in alphabetical order.

Reply:
Done. Thanks for your kind reminder.